build.sh

```diff
@@ -1,48 +1,91 @@
-DIST=$PWD/"dist"
+#!/bin/bash
+ffmpeg_proj_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
-if [ ! -d $DIST ]; then
-    mkdir $DIST
-    echo "CREATED" $DIST
+DIST=${ffmpeg_proj_dir}/"dist"
+
+if [ ! -d ${DIST} ]; then
+    mkdir ${DIST}
```

Comment:
It is good to see ${DIST} when building, even if it exist.
```
mkdir -p ${DIST}
echo "DIST=" ${DIST}
```

Reply:
Made adjustment per this feedback